lib/matplotlib/rcsetup.py

```diff
@@ -501,6 +501,8 @@ def __call__(self, s):
 
 # a map from key -> value, converter
 defaultParams = {
+    'style': ['', validate_stringlist],
```

Comment:
Can you explain a bit more on detail about the recursive dictionary update? I'm willing to do the best implementation possible for this, so if this can be cleaner... 

Envoyé depuis mon appareil Samsung

-------- Message d'origine --------
De : Tony S Yu <notifications@github.com> 
Date : 22/06/2015  06:58  (GMT+01:00) 
À : matplotlib/matplotlib <matplotlib@noreply.github.com> 
Cc : Gilles Marin <gillesmarin1706@gmail.com> 
Objet : Re: [matplotlib] Style RC parameter (#4240) 

In lib/matplotlib/rcsetup.py:
> @@ -501,6 +501,8 @@ def __call__(self, s):
>  
>  # a map from key -> value, converter
>  defaultParams = {
> +    'style': ['', validate_stringlist],

I think you're right, actually: Combining multiple stylesheets in your default matplotlibrc file will probably be a common use case.

That said, I think the implementation might be a bit clearer with a different approach. Have you thought of writing this using recursive dictionary updates instead of looping over parent styles? I've played around with an alternate implementation if you're open to the idea.

—
Reply to this email directly or view it on GitHub.